pom.xml

```diff
@@ -131,6 +131,16 @@
       <id>repo.jenkins-ci.org</id>
       <url>http://repo.jenkins-ci.org/public/</url>
     </repository>
+    <repository>
+      <id>twitter</id>
+      <url>http://maven.twttr.com</url>
```

Comment:
the library isn't a problem, but we have jenkins and central repos only  - and a mirror etc, this ads another dep - it really shoudl be in central